independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html

```diff
@@ -255,7 +255,7 @@ <h1>You just made a Quarkus application.</h1>
                     <p>App configuration: <code>src/main/resources/{config.file-name}</code></p>
                     <p>Static assets: <code>src/main/resources/META-INF/resources/</code></p>
                     <p>Code: <code>{language.dir.code}</code></p>
-                    <p>Dev UI V1: <a href="/q/dev-v1/">/q/dev-v1</a></p>
+                    <p>Dev UI: <a href="/q/dev-ui/">/q/dev-ui</a></p>
```

Comment:
The Dev UI link is already present above. Let's drop this altogether. I will fine tune this and get this puppy home.